packages/netstring/test/test-netstring.js

```diff
@@ -119,9 +155,25 @@ test('concurrent writes', async t => {
     [encoder.encode(''), encoder.encode('A'), encoder.encode('hello')],
     await read(makeNetstringReader(array)),
   );
-});
+};
+test('concurrent writes', concurrentWrites);
+test('concurrent writes (chunked)', concurrentWrites, { chunked: true });
 
-test('round-trip varying messages', async t => {
+const chunkedWrite = async (t, opts) => {
+  const { array, writer } = makeArrayWriter(opts);
+  const strChunks = ['hello', ' ', 'world'];
+  await writer.next(strChunks.map(strChunk => encoder.encode(strChunk)));
```

Comment:
👀 Apparent signature change from bytes to array of byte arrays.

Reply:
Correct, `chunkedWrite` tests the new chunks receiving mode.